code/datums/chatmessage.dm

```diff
@@ -166,6 +173,7 @@ GLOBAL_LIST_INIT(job_colors_pastel, list(
 	else if (extra_classes.Find("emote"))
 		var/image/r_icon = image('icons/UI_Icons/chat/chat_icons.dmi', icon_state = "emote")
 		LAZYADD(prefixes, "\icon[r_icon]")
+		tgt_color = COLOR_CHAT_EMOTE
```

Comment:
```suggestion
	else if (extra_classes.Find("emote"))
		var/image/r_icon = image('icons/UI_Icons/chat/chat_icons.dmi', icon_state = "emote")
		LAZYADD(prefixes, "\icon[r_icon]")
		tgt_color = COLOR_CHAT_EMOTE
```
if we end up keeping static emote color this should be before `isliving()` checks and should skip them.